package.json

```diff
@@ -18,58 +18,44 @@
   "homepage": "https://github.com/freeCodeCamp/freeCodeCamp#readme",
   "author": "freeCodeCamp <team@freecodecamp.org>",
   "main": "none",
-  "workspaces": [
```

Comment:
I cannot comment further. Line 9:

Should we remove/change `engines.npm` to `engines.pnpm`?

Reply:
Thanks, I'll do that.